Libraries/LibWeb/SVG/SVGGraphicsElement.cpp

```diff
@@ -266,13 +267,10 @@ Optional<float> SVGGraphicsElement::stroke_opacity() const
     return layout_node()->computed_values().stroke_opacity();
 }
 
-Optional<float> SVGGraphicsElement::stroke_width() const
+Optional<float> SVGGraphicsElement::resolve_relative_to_viewport_size(CSS::LengthPercentage const& length_percentage) const
```

Comment:
(I made the body of what previously `stroke_width()` a new helper that's called by both `stroke_dashoffset` and `stroke_width).)